### PR TITLE
chore(ui): add optional right sidebar in preparation for feedback

### PR DESCRIPTION
**Pull request by gtarpenning**
## Description

This pr:
- renames `SplitPanel` -> `SplitPanelLeft`
- refactors calculations in `SplitPanelLeft` for performance, including a 60fps throttle
- adds `SplitPanelRight` which is very similar, but allows for a right drawer, and doesn't have background colors.

This pr should be a no-op without the human feedback stuff. 

Example with human feedback drawer: 
slider2

## Testing

👀

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx [hunk 1]
```diff
@@ -305,8 +305,8 @@ const CallPageInnerVertical: FC<{
           </Box>
         </Box>
       }
-      isSidebarOpen={showTraceTree}
       headerContent={<CallOverview call={currentCall} />}
+      isLeftSidebarOpen={showTraceTree}
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx:309]: more clear ordering

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx [hunk 2]
```diff
@@ -244,67 +249,101 @@ export const SimplePageLayoutWithHeader: FC<{
...
+              }
+            />
           }
         />
       </div>
     </Box>
   );
 };
 
+const MainPeekContent: FC<{
```
Comment (tssweeney) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx:284]: nit: This view is not just for peeking. It is visible when expanded. Suggest renaming to something like `SimpleTabView` to more accurately reflect the purpose

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,224 @@
+/**
```
Comment (tssweeney) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx:1]: hmmm ... how much of this is a copy of `weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SplitPanelRight.tsx`? Is there a way to make this generic? ... ie: `SplitPanel` that has a left/right param?

Comment (gtarpenning) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx:1]: I tried, but there are a ton of specific calculations that made the combo component hard to understand. I can try again though, it worked, was just complicated.

Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx:1]: Ok. It is possible that generalizing makes it harder to maintain or easier.... , up to you. i would at the very least make a comment saying that this is a sister to the other component. Or even put them in the same file / subdir to emphasize their similarity. Furthermore, probably rename this one `SplitPanelLeft`

Comment (gtarpenning) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/TraceTreeSplitPanel.tsx]: Yeah agreed, done

**Review by tssweeney — APPROVED** [review]

**Pull request merged**